contracts/p1/BasketHandler.sol

```diff
@@ -138,6 +139,19 @@ contract BasketHandlerP1 is ComponentP1, IBasketHandler {
     // and everything except redemption should be paused.
     bool private disabled;
 
+    // These are effectively local variables of _switchBasket.
+    // Nothing should use their values from previous transactions.
+    EnumerableSet.Bytes32Set private _targetNames;
+    Basket private _newBasket; // Always empty
```

Comment:
Good call to move these up. I was adding my new variables down at the bottom by the storage gap, but this will make it easier to track all variables and make sure we don't accidentally insert a new variable in the middle.

Reply:
I think its safer and its kind of the norm to have all variables on top. Felt weird to have them at the bottom. I think when people read contracts you do not expect variables at the bottom. 👍